tests/end-to-end/backup-restore-test/backupe2e/service-catalog/brokers_flow.go

```diff
@@ -0,0 +1,372 @@
+package service_catalog
+
+import (
+	"github.com/sirupsen/logrus"
```

Comment:
sort import, right now you have groups that don't make sense


same suggestion for rest